tests/testthat/test_deprecated.R

```diff
@@ -0,0 +1,12 @@
+library(testthat)
+library(dials)
```

Comment:
no need to library these

Reply:
`testhat`/`dials` library calls removed from `test_deprecated.R` and all other test files.